src/lsm/manifest_level_fuzz.zig

```diff
@@ -385,7 +390,7 @@ fn EnvironmentType(comptime table_count_max: u32, comptime node_size: u32) type
                 const env_table = env.find_exact(level_table);
                 assert(level_table.equal(env_table));
 
-                env.level.set_snapshot_max(env.snapshot, env_table);
+                env.level.set_snapshot_max(env.snapshot, .{.copy=env_table});
```

Comment:
Weird that `zig fmt` didn't fix this
```suggestion
                env.level.set_snapshot_max(env.snapshot, .{.copy = env_table});
```